CHANGELOG.md

```diff
@@ -68,17 +68,29 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Middleware support (for e.g. for centrally handling authorization).
 
 ## [1.1.0] - 2019-05-30
```

Comment:
I would add a link to a full list of changes for every release section.
For example for 1.1.0:
```
## [1.1.0] - 2019-05-30

### Added

- Travis builds for tags.

[1.1.0]: https://github.com/tarantool/http/compare/1.0.6...1.1.0
```

Rendered markdown will convert 1.1.0 to a link to a full list with changes. I found it is convenient for everyone who will read changelog. It's my own opinion and feel free to ignore :)